src/coffee/fetchstocks.coffee

```diff
@@ -8,7 +8,8 @@ class FetchStocks
 
 
   run: ->
-    query = @client.inventoryEntries.all().sort('sku').expand('supplyChannel')
+    # Do NOT use custom sorting due to https://github.com/sphereio/sphere-node-sdk/issues/194
```

Comment:
I am not sure we want to add this comments to the code, should be added to the commit or the PR